### PR TITLE
Fixed some typos

### PR DESCRIPTION
**Pull request by dudemelo**

### CODE REVIEW

File: src/main/resources/rulesets/cleancode.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@ class Foo
           externalInfoUrl="http://phpmd.org/rules/cleancode.html#staticaccess">
         <description>
             <![CDATA[
-Static access causes inexchangable dependencies to other classes and leads to hard to test code. Avoid
+Static access causes unexchangable dependencies to other classes and leads to hard to test code. Avoid
```
Comment (ravage84) [inline src/main/resources/rulesets/cleancode.xml]: "unexchangeable" seems to be correct.

File: src/site/rst/documentation/ant-task.rst [hunk 2]
```diff
@@ -97,8 +97,8 @@ is a modified version of the previous example: ::
       </phpmd>
   </target>
 
-Postprocessing the report file with XSLT
```
Comment (ravage84) [inline src/site/rst/documentation/ant-task.rst]: I think either Post-processing or postprocessing ist OK.

File: src/main/resources/rulesets/cleancode.xml (hunk 1 above)
Comment (dudemelo) [inline src/main/resources/rulesets/cleancode.xml]: Yep, thx!

File: src/site/rst/documentation/index.rst [hunk 3]
```diff
@@ -24,8 +24,8 @@ You can pass a file name or a directory name containing PHP source
 code to PHPMD.
 
 The PHPMD Phar distribution includes the rule set files inside
-its archive, even if the "rulesets/codesize.xml" parameter above looks 
-like a filesystem reference.
```
Comment (ravage84) [inline src/site/rst/documentation/index.rst]: Filesystem seems to be valid.

**Comment by ravage84** [comment]
Many thanks for the various fixes.
If you can address my comments, we can get this merged.

File: src/site/rst/documentation/ant-task.rst (hunk 2 above)
Comment (dudemelo) [inline src/site/rst/documentation/ant-task.rst]: Yes, we can write with or without a hyphen after the prefix... i just let my spellchecker do its job. :smile:

**Comment by dudemelo** [comment]
Done @ravage84 !

File: src/site/rst/documentation/ant-task.rst [hunk 4]
```diff
@@ -98,7 +98,7 @@ is a modified version of the previous example: ::
   </target>
 
 Postprocessing the report file with XSLT
-========================================
+=========================================
```
Comment (ravage84) [inline src/site/rst/documentation/ant-task.rst]: This one is now too long

**Comment by ravage84** [comment]
Merged and fixed outstanding issue in https://github.com/phpmd/phpmd/commit/2b9c2417a18696dfb578b38c116cd0ddc19b256e

**Comment by ravage84** [comment]
Again, :beer:

**Pull request merged**